packages/react-core/src/components/DescriptionList/DescriptionListDescription.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/DescriptionList/description-list';
+import { css } from '@patternfly/react-styles';
+
+export interface DescriptionListDescriptionProps extends React.HTMLProps<HTMLElement> {
+  /** Anything that can be rendered inside of list description */
+  children: React.ReactNode;
+  /** Additional classes added to the DescriptionListDescription */
+  className?: string;
+  /** Indicates which component will be used for the text wrapped by the description */
+  textComponent?: React.ReactNode;
+  /** Default hyperlink location */
+  href?: string;
```

Comment:
this is not needed.  the anchor would be. A child of the 'pf-c-description-list__text' `div` like it is in core.